airflow/www/static/js/grid/FilterBar.jsx

```diff
@@ -0,0 +1,121 @@
+/*!
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+/* global filtersOptions */
+
+import {
+  Box,
+  Button,
+  Flex,
+  Input,
+  Select,
+} from '@chakra-ui/react';
+import React from 'react';
+
+import useFilters from './utils/useFilters';
+
+const FilterBar = () => {
+  const {
+    filters,
+    onBaseDateChange,
+    onNumRunsChange,
+    onRunTypeChange,
+    onRunStateChange,
+    onTaskStateChange,
+    clearFilters,
+  } = useFilters();
+
+  const inputStyles = { backgroundColor: 'white', size: 'lg' };
+
+  return (
+    <Flex backgroundColor="#f0f0f0" mt={0} mb={2} p={4}>
+      <Box px={2}>
+        <Input
+          {...inputStyles}
+          type="datetime-local"
+          value={filters.baseDate || ''}
```

Comment:
The displayed date here isn't updating when a user changes their timezone. We need to format the date with `const { timezone } = useTimezone();`

Reply:
Good catch. I just updated the pull request to handle the timezone properly :)